ner_v1/detectors/textual/name/lang_constants.py

```diff
@@ -113,7 +113,88 @@
            'अपनी मा को जा चूस', 'भें के लॉड', 'भें के टक्के: गो आंड सक युवर सिस्टर’स बॉल्', 'अबला नारी तेरा बुबले भारी',
```

Comment:
1. Can you please fix lint errors in this file?
2. Please use one style quotes for string literals for the whole file - either `''` or `""` for consistently